libs/common/src/key-management/master-password/abstractions/master-password.service.abstraction.ts

```diff
@@ -44,33 +62,67 @@ export abstract class MasterPasswordServiceAbstraction {
     userId: string,
     userKey?: EncString,
   ) => Promise<UserKey | null>;
+
+  /**
+   * Makes the authentication hash for authenticating to the server with the master password.
+   */
```

Comment:
```suggestion
  /**
   * Makes the authentication hash for authenticating to the server with the master password.
   * @param password The master password.
   * @param kdf The KDF configuration.
   * @param salt The master password salt to use. See {@link saltForUser$} for current salt.
   * @throws If password, KDF or salt are null or undefined.
   */
```

⛏️  Would be helpful to link to `saltForUser$` here IMO.